test/verify/files/cockpitMachinesTestExternalProvider.js

```diff
@@ -173,6 +176,18 @@
     },
 
     /**
+     * Call `Send Non-Maskable Interrupt` on the VM.
+     */
+    SENDNMI_VM: function (payload) {
+      console.log('PROVIDER.SENDNMI_VM called with params: ' + JSON.stringify(payload));
```

Comment:
Please forgot my previous comment regarding logDebug(). It's not available in the _testing_ external provider since it's written in vanilla JS without additional libraries. I missed the filename when reviewing.

Since this code is meant for either integration test of integration with an _external provider_ or as an example of it's use, I think the console.log() can be here as it is - this code will not be executed on production.